src/resources/i18n/translations/en.json

```diff
@@ -432,10 +432,10 @@
   "No Wifi Connection": "No Wi-Fi connection",
   "No Wifi Only": "No, Wi-Fi only",
   "Now Playing": "Now Playing",
-  "Now Playing Info": "\"Now Playing\" Info",
+  "Now Playing Info": "\"Now Playing\" info",
```

Comment:
Not 100% sure here.

Reply:
This is actually an oversight. We now call this section "Episode Summary" rather than "Now Playing Info". I'll remove "Now Playing Info" altogether.